Gems/PhysX/Code/Source/JointComponent.h

```diff
@@ -64,6 +64,8 @@ namespace PhysX
             const JointLimitProperties& limitProperties,
             const JointMotorProperties& motorProperties);
 
+        AZStd::string GetJointName() const;
```

Comment:
suggestion: We could have this return a `const AZStd::string&` as we're not making this virtual and might have to deal with users returning a temporary. Returning by reference will be faster in this case.